api/v1beta1/ovncontroller_types.go

```diff
@@ -158,7 +158,7 @@ type OVSExternalIDs struct {
 
 	// +kubebuilder:validation:Optional
 	// +kubebuilder:default=true
-	EnableChassisAsGateway bool `json:"enable-chassis-as-gateway"`
+	EnableChassisAsGateway *bool `json:"enable-chassis-as-gateway,omitempty"`
```

Comment:
I agree that it needs to be a pointer as you want to make it optional. I'm not sure you need the omitempty after you made it to a pointer.

Reply:
thx dropped it, as seems not needed.